### ISSUE TITLE
Delete old podcasts

### ISSUE THREAD

**Issue by fidegr** [LibreTime/libretime#1055]
Hello,

I suggest that in the podcast section you can select how many to keep and that the oldest ones are automatically deleted.
I don't know what policy there is right now to keep or not the old episodes of the podcast.

Thanks, salute!

**Comment by Robbt** [comment]
Yeah just to be clear there is no autodelete for podcasts currently. You have to manually delete them until this functionality is added.

**Comment by fidegr** [comment]
Ok, as a temporary solution I can think of adding this task to crontab

 find /srv/airtime/ stor/imported/ -type f -mtime +14 -size + 20M -exec rm {} \;

This will erase all files of more than 20 Megabytes, which I understand are not jingles or music themes, more than 14 days old.

Can it be troublesome?

**Comment by paddatrapper** [comment]
You may need to update the DB too

**Comment by fidegr** [comment]
Ok, how would it be to update the database?

Thanks in advance for the help

**Comment by paddatrapper** [comment]
The database is Postgres, so you would need to run SQL queries against it. You can see a listing of tables and columns in #958

**Comment by Robbt** [comment]
Yeah basically the files would still show up in your listings but they wouldn't work if you tried to add them and they could cause problems with playout, like LibreTime thinks that they are real and then when it tries to play the file you have dead air. I would recommend against doing the bulk deletion unless you are really short on time and disk space. Ultimately it will cost you more time as well going and manually deleting them. We should make an auto-delete option at some point but it isn't at the top of the priority list for anyone as far as I know.
